recipes/ablang2/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
```

Comment:
Please remove this file. Packages on conda-forge should only depend on conda-forge packages.